cartography/data/indexes.cypher

```diff
@@ -455,3 +455,8 @@ CREATE INDEX IF NOT EXISTS FOR (n:KubernetesSecret) ON (n.lastupdated);
 CREATE INDEX IF NOT EXISTS FOR (n:KubernetesService) ON (n.id);
 CREATE INDEX IF NOT EXISTS FOR (n:KubernetesService) ON (n.name);
 CREATE INDEX IF NOT EXISTS FOR (n:KubernetesService) ON (n.lastupdated);
+CREATE INDEX IF NOT EXISTS FOR (n:WAFv2WebACL) ON (n.id);
+CREATE INDEX IF NOT EXISTS FOR (n:WAFv2WebACL) ON (n.name);
+CREATE INDEX IF NOT EXISTS FOR (n:WAFv2WebACL) ON (n.lastupdated);
+CREATE INDEX IF NOT EXISTS FOR (n:WAFv2APIRule) ON (n.name);
```

Comment:
Also generate some kind of unique if for this node. If the rules don't have their own arns, generate one. Maybe like `<arn of acl>/rule-name`